README.md

```
@@ -1 +1,185 @@
+  end
+}
+```
+
+### [vim-plug](https://github.com/junegunn/vim-plug)
+
+```vim
+Plug 'VatsalSy/gruvbox_custom_crisp_themes.nvim'
+```
+
+Then in your config:
+```vim
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)